src/processors/CropOnMarkers.py

```diff
@@ -30,7 +31,7 @@ def __init__(self, marker_ops, cwd):
         self.marker_rescale_steps = marker_ops.get("marker_rescale_steps", 10)
         self.apply_erode_subtract = marker_ops.get("apply_erode_subtract", 1)
         if not os.path.exists(self.marker_path):
-            print(
+            logger.error(
```

Comment:
now how about removing the "Error: " prefix?

Reply:
fixed